src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts

```diff
@@ -123,34 +132,40 @@ const shuffle = ({ file, blockWidth }: shuffleArgs) => {
     }
 
     const totalBlocksNum = (w * h) / (blockWidth * blockWidth) // this will be a whole number, because we resize earlier
-    const permutations: number[] = []
+    // const permutations: number[] = []
     const img = file.bitmap.data
+    const seed = String(Math.random()) // ! how many states can this generate?
```

Comment:
a lot. 0 - 9 for each decimal place digit & 16 decimal places => 10 ^ 16 states of the prng. That means, that log base 2 of 10 ^ 16 = 53.15. So the prng can generate 2 ^ 53.15 states. Solid